osquery/tables/networking/windows/windows_firewall_rules.h

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <netfw.h>
+
+namespace osquery {
+namespace tables {
+
+struct FirewallRule {
```

Comment:
is_standard_layout/is_trivial types should be default-initialized

Reply:
btw plenty of code where this is not done, example
https://github.com/osquery/osquery/blob/2e1fff970274d875cfe345b6534d29a0c6397d58/osquery/utils/windows/shelllnk.h

Reply:
> btw plenty of code where this is not done, example https://github.com/osquery/osquery/blob/2e1fff970274d875cfe345b6534d29a0c6397d58/osquery/utils/windows/shelllnk.h

Yes, those structures should be fixed too (not in this PR)